fhir-model/src/main/java/com/ibm/fhir/model/config/FHIRModelConfig.java

```diff
@@ -39,10 +39,16 @@
      */
     public static final String PROPERTY_CHECK_REFERENCE_TYPES = "com.ibm.fhir.model.checkReferenceTypes";
 
+    /**
+     * Used to determine whether CodeableConcepts that don't contain both system and code are checked during object construction
+     */
+    public static final String PROPERTY_ADVANCED_CODEABLE_CONCEPT_VALIDATION = "com.ibm.fhir.model.advancedCodeableConcepValidation";
```

Comment:
does it also control whether we do syntax-based validation of UCUM and Language valuesets?

Reply:
Yes. I've updated the prologue to reflect that.